README.md

```diff
@@ -24,6 +24,10 @@ If you want to reactify modules with other extensions, pass an `-x /
 
     % browserify -t coffeeify -t [ reactify --extension coffee ] main.coffee
 
+If you want to specify your own extensions, pass an `--extensions` option:
+
+    % browserify -t [ reactify --extensions jsx,coffee ] main.jsx
```

Comment:
No, you can't have spaces. I suppose the code could strip the spaces for the individual extensions, but using spaces in command line arguments is bad form anyways.